src/main_controller.ts

```diff
@@ -119,6 +120,13 @@ export class MainController implements vscode.Disposable {
         if (!config.clean && config.neovimInitPath) {
             args.push("-u", config.neovimInitPath);
         }
+        if (config.afterInitConfig.length) {
+            const storagePath = this.extContext.globalStorageUri.fsPath;
+            if (!existsSync(storagePath)) mkdirSync(storagePath, { recursive: true });
+            const vim = path.join(storagePath, "afterInitConfig.vim");
+            writeFileSync(vim, config.afterInitConfig, { encoding: "utf8" });
```

Comment:
Writing a file should not be needed. Nvim has `nvim_exec2`, or you can `:source` a temporary buffer.

Reply:
Yes, I seem to have made it complicated (misled by the previous requirement of startup parameters in the #1350).